src/main/scala/org/broadinstitute/dsde/workbench/leonardo/auth/DefaultServiceAccountProvider.scala

```diff
@@ -29,4 +29,6 @@ class DefaultServiceAccountProvider(config: Config) extends ServiceAccountProvid
   override def listUsersStagingBucketReaders(userEmail: WorkbenchEmail)(implicit executionContext: ExecutionContext): Future[List[WorkbenchEmail]] = {
     Future.successful(List.empty)
   }
+
+  override def getAccessToken(userEmail: WorkbenchEmail, googleProject: GoogleProject): String = ""
```

Comment:
We should make sure `DefaultServiceAccountProvider` is actually usable. I feel like returning "" would cause exceptions to be thrown in LeoService?

That said, I'm not entirely sure how to handle this. I don't think we can easily get a token for the compute engine default service account. Maybe this method should actually be returning `Future[Option[String]]` and if it's `None` we just skip bucket validation.